blokus_env/blokus_env.py

```diff
@@ -116,8 +140,8 @@ def render(self, mode='human'):
         Returns:
         - img (np.ndarray): Rendered image if mode is 'rgb_array'.
         """
-        img = self.renderer.render(self.board, self.pieces)
-        return img
+        self.renderer.render(self.board, self.pieces)
```

Comment:
Please allow render() to return an image. This is important for the RL code to be able to record videos correctly

Reply:
Ok! :+1: